packages/server/dataloader/atlassianLoaders.ts

```diff
@@ -230,8 +230,9 @@ export const jiraIssue = (
             const possibleEstimationFieldNames = [] as string[]
             Object.entries<{schema: {type: string}}>(issueRes.editmeta?.fields)?.forEach(
               ([fieldId, {schema}]) => {
-                if (isValidEstimationField(schema.type, issueRes.names[fieldId], fieldId)) {
-                  possibleEstimationFieldNames.push(issueRes.names[fieldId])
+                const fieldName = issueRes.names[fieldId]
```

Comment:
+1 Fine for now, but I would have assumed that without name, Jira uses the id for it. Thus I would replace this line with
```
const fieldName = issueRes.names[fieldId] ?? fieldId
```

Reply:
Fixed